src/terraformCloud/index.ts

```diff
@@ -48,6 +48,7 @@ export const apiClient = new Zodios(baseUrl, [
 ]);
 apiClient.use(jsonHeader);
 apiClient.use(userAgentHeader);
+apiClient.use(pluginLogger());
```

Comment:
This was kinda forgotten here, but also I found it useful for debugging, I'm not sure what to do with it. Maybe remove the `use()` part and leave the plugin logic in place?

I suppose we don't want to keep logging every request in prod.